GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2981,7 +2981,7 @@ private void LayoutRevisionInfo()
             }
             else
             {
-                int width = DpiUtil.Scale(420);
+                int width = DpiUtil.Scale(490) + SystemInformation.VerticalScrollBarWidth;
```

Comment:
What's this magic number? Worth having at least a comment about it.
I know there was another undocumented magic number, but since you changed it, you probably have some idea what it is.

Reply:
ok